### PR TITLE
Feature/ idva5 269 check your answers before sending your application

### PR DESCRIPTION
**Pull request by hemerem-ch**
Build check your answer page:
- [ ] Controller
- [ ] njk page
- [ ] Test

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/controllers/checkYourAnswers.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+export const post = async (req: Request, res: Response, next: NextFunction) => {
+    const lang = selectLang(req.query.lang);
+    const locales = getLocalesService();
+    const errorList = validationResult(req);
+    const previousPage: string = addLangToUrl(BASE_URL + HOW_IDENTITY_DOCUMENTS_CHECKED, lang);
+    const currentUrl: string = BASE_URL + CHECK_YOUR_ANSWERS;
+    const session: Session = req.session as any as Session;
+    const clientData: ClientData = session.getExtraData(USER_DATA) ? session.getExtraData(USER_DATA)! : {};
+
+    const formattedAddress = FormatService.formatAddress(clientData.address);
```
Comment (sampankumar) [inline src/controllers/checkYourAnswers.ts:57]: You can do all of this if there is any error. If there is no error then you dont need to get this data

**Review by sampankumar — APPROVED** [review]

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**